### PR TITLE
Linking my blog 

### PR DESCRIPTION
**Pull request by hshafaq7**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -258,3 +258,11 @@
...
   author_name: Yash Kothari
 
+- url: https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
+  title: Here I come Coding In
+  img: haider1.jpg
+  author_name: Haider Shafaq
+
+
+
+
```
Comment (nikit19) [inline _data/blogs.yml]: Please remove these blank lines

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please ensure that the dimensions of your image is 240 x 240 pixels.
Please upload only 1 image.

File: _data/blogs.yml [hunk 2]
```diff
@@ -258,3 +258,11 @@
   img: Kothari.jpg
   author_name: Yash Kothari
 
+- url: https://mygeekyworld786.wordpress.com/2017/12/30/here-i-come-coding-in/
+  title: Here I come Coding In
+  img: haider1.jpg
+  author_name: Haider Shafaq
+
+
```
Comment (harshit98) [inline _data/blogs.yml]: Remove extra lines.

**Review by isuruAb — COMMENTED** [review]
please do following changes
- upload a 240x 240 sized image
- remove extra image
- rotate your image 180 degrees
- resolve conflicts

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
See the reviews @hshafaq7

**Pull request closed without merging**